canonical/ubuntu.com
# Core 24 release copy updates

_Go live date June 3rd at 6PM CET as agreed on MM_
## Done

- Applied copy changes as per [doc](https://docs.google.com/document/d/1Sxdl5cGGe1wbHmKB2eL2zQLfVW3KBea02a2FIGx3VdQ/edit) and [design](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=1-72&mode=dev)
- Updated releases.yaml to reflect new release

## QA

- View the site locally in your web browser at: https://ubuntu-com-13847.demos.haus/download/core
- See that the requested changes have been made

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10919
